coding-standards/RegEx.md

```diff
@@ -0,0 +1,57 @@
+# Standard regular expressions to be used across the technologies
+
+1. **Phone number without country code**:
+   - Regex: `^\d{6,14}$`
+   - Example HTML:
+     ```html
+     <input type="tel" name="phone" pattern="^\d{6,14}$" required>
```

Comment:
Yes, we don't recommend using these attributes in HTML, however - used these attributes in example so that the usage can be explained in a single line of code.

Anyway, added a note at the top of the document to convey that these are just for reference and we don't recommend using these attributes directly in HTML.